### ISSUE TITLE
Polish starts of the Understanding documents

### ISSUE THREAD

**Issue by alastc** [w3c/wcag21#882]
From an onlist discussion, @mbgower suggested using his [Simple Statements of new WCAG 2.1 SCs](https://www.ibm.com/blogs/age-and-ability/2018/02/08/simplifying-new-wcag-2-1-guidelines/) as the start of the intent section.

I think it's best to do towards the end of the understanding doc editing process, to  prevent clashes.

Also, it should be done on a case-by-case basis, some of the docs may have very well thought-out introductions. 

Adding this issue for tracking.

**Comment by michael-n-cooper** [comment]
This issue was moved to w3c/wcag#395
